test/0500-remove-index-patterns-without-uid

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# test init script properly removes desired kibana index patterns
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+os::util::environment::use_sudo
+
+test_name=$(basename $0)
+os::test::junit::declare_suite_start ${test_name}
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+fi
+
+cleanup() {
+    local return_code="$?"
+    set +e
+    if [ $return_code = 0 ] ; then
+        mycmd=os::log::info
+    else
+        mycmd=os::log::error
+    fi
+    $mycmd ${test_name} test finished at $( date )
+    # this will call declare_test_end, suite_end, etc.
+    os::test::junit::reconcile_output
+    exit $return_code
+}
+trap "cleanup" EXIT
+
+seed_kibana_data(){
+  local pod=$1
+  local uuid=${2:-}
+  os::log::info Seeding 20 Kibana index-patterns ${uuid:-without uuid}
+  names=(abc123 xyz123 gef123 kjl123 zzz123)
+  for n in "${names[@]}"
+  do
+    for i in $(seq 1 4)
+    do
+      doc=".kibana.$n/index-pattern/project.foo${i}${uuid}.*"
+      result=$(oc exec -c elasticsearch $pod -- es_util --query=$doc -XPOST -d '{"key":"value"}')
+      if [ $? -ne 0 ] ; then
+        os::log::error Error seeding index-patterns
+        exit 1
+      fi
+    done
+  done
+}
+
+os::log::info Starting ${test_name} test at $( date )
+
+espod=$( get_es_pod es )
+cmd="oc exec -c elasticsearch -n openshift-logging $espod"
```

Comment:
use `$LOGGING_NS` instead of `openshift-logging`

Reply:
here forward we will never deploy to an alternate namespace.  Do we need to parameterize it?

Reply:
We thought we would never deploy to an alternate to `logging` too . . . it is parameterized everywhere else, should also be here too for the sake of consistency